src/effects/native/flangereffect.cpp

```diff
@@ -28,26 +31,60 @@ EffectManifest FlangerEffect::getManifest() {
     period->setId("period");
     period->setName(QObject::tr("Period"));
     period->setDescription(QObject::tr("Controls the period of the LFO (low frequency oscillator)\n"
-        "1/4 - 4 beats rounded to 1/2 beat if tempo is detected (decks and samplers) \n"
-        "0.05 - 4 seconds if no tempo is detected (mic & aux inputs, master mix)"));
+        "1/4 - 20 beats rounded to 1/2 beat if tempo is detected (decks and samplers) \n"
+        "0.05 - 20 seconds if no tempo is detected (mic & aux inputs, master mix)"));
     period->setControlHint(EffectManifestParameter::ControlHint::KNOB_LINEAR);
     period->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
     period->setUnitsHint(EffectManifestParameter::UnitsHint::BEATS);
     period->setMinimum(0.00);
-    period->setMaximum(4.00);
+    period->setMaximum(20.00);
```

Comment:
20 beats? Why? How does that make sense musically? How about 16 beats? Maybe we should go back to a logarithmic scale for this parameter.

Reply:
I do not mid about the maximum.
20 just marches our original version @96 kHz. Hardware flangers have a Max around 10 s that's 20 for a 120 BPM track.

Real flangers have also a speed (fast fully right) parameter with a Hz scale. I like this when using, but it does not match our other effects.

How about that? @sohet?

Reply:
This is synced to the tempo, so the time in seconds is irrelevant. The maximum is important because turning the parameter to the maximum should be useful somehow. I can't think of a musical context in which a 20 beat period would make sense. A maximum of 16 beats or 4 bars with a 4/4 time signature seems to be a reasonable maximum. Then the user can also know the center of the knob is 8 beats and 1/4 of the knob is 4 beats (or 1 bar in 4/4).

Reply:
>20 beats? Why? How does that make sense musically? How about 16 beats? Maybe we should go back to a logarithmic scale for this parameter.

I don't want the max value to be shorter than the current 20. How about 32 and logarithmic scale?

>Real flangers have also a speed (fast fully right) parameter with a Hz scale. I like this when using, but it does not match our other effects. How about that?

I also like the way.

Reply:
Is this always going to be in beats rather than hertz where a beat is detected? It's often musically desirable to have an effect that is not synced to the tempo, to create tension

Reply:
@christophski I tried adding a button parameter to disable tempo syncing in #1257 but I did not think it was useful. For Echo, yes it is helpful, but I don't think it is for Flanger.